packages/vue-components/src/Searchbar.vue

```diff
@@ -89,6 +89,11 @@ export default {
   },
   data() {
     return {
+      /*
+       * "value" is passed down as a prop. In order to not mutate props (causes Vue warning),
```

Comment:
hmm could we change `value` to a `data` property instead?

dosen't seem like a prop https://markbind-master.netlify.app/userguide/makingthesitesearchable#search-bars

Reply:
Hmm yeah that's true. Changed it accordingly!